src/firebase/notes.js

```diff
@@ -0,0 +1,34 @@
+import { fireStoreRef } from '.';
+import firebase from 'react-native-firebase';
+
+export async function addNoteForTalk(talkId, note = 'hello world') {
+  var user = firebase.auth().currentUser;
```

Comment:
auth is already exported in index.js inside firebase folder, so we can import auth from there.

Reply:
Used auth from exported function.